_posts/2021-02-1-pulp-community-update.md

```diff
@@ -0,0 +1,102 @@
+* Added support for repairing Pulp by detecting and redownloading missing or corrupted artifact files.
+* Users can add checksums to `ALLOWED_CONTENT_CHECKSUMS` by allowing them to populate checksums with the `handle-artifact-checksums` command.
+
+In tech preview in Pulpcore 3.9:
+
+* Debugging has been made easier with a correlation id that can either be autogenerated or passed in with a Correlation-ID header.
+
+For a full list of features and bug fixes, check out the [changelog](https://docs.pulpproject.org/pulpcore/en/3.9/changes.html).
+
+The [3.9.1 release](https://docs.pulpproject.org/pulpcore/en/3.9.1/changes.html#id1) contained a fix for a bug that was encountered by users that set a custom CHUNKED_UPLOAD_DIR setting. The value for this setting must now be a relative path inside MEDIA_ROOT.
+
+[Pulp Certguard](https://docs.pulpproject.org/pulp_certguard/changes.html#id1) and [Pulp File](https://docs.pulpproject.org/pulp_file/changes.html) compatibility releases were made for Pulpcore 3.9 and 3.10.
+
+## Pulp Python 3.0.0
+
+The long-awaited update to the Pulp Python plugin is now generally available. This release adds substantial enhancements to managing your Python content in Pulp. Over twelve beta releases, functionality was added so you can mirror the whole of PyPI in just under one hour, as well as the ability to upload your own Python content, among many other changes. To get started and learn more about the Pulp Python workflows, see the [Pulp Python documentation](https://pulp-python.readthedocs.io/en/latest/).
+
+## Pulp 2-to-3 Migration 0.6.0
+
+Pulp 2 will sunset in 2022. If you haven't started planning your [migration](https://pulpproject.org/migrate-to-pulp-3/) yet, there's never been a better time.
+
+In this release, there is a tech preview available for migrating Debian packages.
+As well as this, this release adds a reset endpoint. If you want, you can now start a migration from scratch.
+You can also skip corrupted or missing Pulp 2 content.
+
+If you're interested in learning more about migrating to Pulp 3, take a look at the [migration documentation](https://pulp-2to3-migration.readthedocs.io/en/latest/index.html).
+
+## Pulp Container 2.2.0
+
+As well as a number of bug fixes, this release adds the ability to define namespaces for container group repositories and distributions. Uploaded chunked files are now stored in shared storage rather than `/var/lib/pulp,`.
+
+This release is compatible with Pulpcore 3.9 and the future release 3.10.
+
+For more information about Pulp Container and its workflows, see the [Pulp Container documentation](https://docs.pulpproject.org/pulp_container/index.html).
+
+## Pulp Debian 2.8 & 2.9
+
+Some `APT` repositories store a repository root to store their release files. The main feature of this release adds the capability to synchronize repositories using this alternative repository format.
```

Comment:
```suggestion
Some `APT` repositories store their release files in the repository root. The main feature of this release adds the capability to synchronize repositories using this alternative repository format.
```

This is instead of the default `dists/<distribution_name>` location.

Reply:
Ah yeah. I make no sense. That's why I need a cool off period to reread. Thank you!